spec/nameable_spec.rb

```diff
@@ -0,0 +1,13 @@
+require './nameable'
+
+describe Nameable do
+  before :each do
+    @nameable = Nameable.new
+  end
```

Comment:
- [ ] Let ensure that there are unit tests for all the methods in all the classes, please kindly write unit test for the `correct_name` method in the Nameable class.